app/src/main/res/values/strings.xml

```diff
@@ -5002,5 +5002,12 @@
   <string name="AccountDeleted">Account deleted</string>
   <string name="AccountDeletedText">Your account %1$s (%2$s) is now deleted from Telegram servers, and can no longer be restored.\n\nUsing the same phone number will create a new account.</string>
   <string name="ViewOnce">View once</string>
+  <string name="ConvertToBroadcastGroup">Convert to Broadcast Group</string>
+  <string name="xConvertToBroadcastGroupDesc_one">Broadcast groups can have over %1$s members, but only admins can send messages in them.</string>
+  <string name="xConvertToBroadcastGroupDesc_other">Broadcast groups can have over %1$s members, but only admins can send messages in them.</string>
```

Comment:
1. `_one` variant must be in singular form.
2. `x` prefix isn't required, only suffix part is important for plurals.

```suggestion
  <string name="ConvertToBroadcastGroupDesc_one">Broadcast groups can have over %1$s member, but only admins can send messages in them.</string>
  <string name="ConvertToBroadcastGroupDesc_other">Broadcast groups can have over %1$s members, but only admins can send messages in them.</string>
```